backend/api/post/postService.js

```diff
@@ -118,6 +119,14 @@ module.exports = function (app) {
     }
   })
 
+  app.put("/api/posts/:id/moderators", authUserLoggedIn, authRole([roles.POSTOWNER, roles.POSTMODERATOR]), handleModerator, async (req, res) => {
```

Comment:
har testat logga in med användare bob@haha.se (pass: 123).
skapat en post.
vet inte varför men den användaren har endast role.USER så jag kan inte gå vidare med requesten.